include/gz/sim/Util.hh

```diff
@@ -287,19 +287,31 @@ namespace ignition
     std::optional<math::Vector3d> IGNITION_GAZEBO_VISIBLE sphericalCoordinates(
         Entity _entity, const EntityComponentManager &_ecm);
 
-    /// \brief Environment variable holding resource paths.
+    /// \brief Environment variable holding resource paths. Prefer using
+    /// GZ_SIM_RESOURCE_PATH for compatibility with newer versions of Gazebo.
     const std::string kResourcePathEnv{"IGN_GAZEBO_RESOURCE_PATH"};
+    /// \brief Environment variable holding resource paths. 
+    const std::string kResourcePathEnvGzSim{"GZ_SIM_RESOURCE_PATH"};
```

Comment:
Not actionable here, but we should probably switch these over to `constexpr const char*` at some point.

Reply:
Agreed!